cmd/cloud/organization.go

```diff
@@ -0,0 +1,69 @@
+package cloud
+
+import (
+	"io"
+
+	"github.com/spf13/cobra"
+
+	"github.com/astronomer/astro-cli/cloud/organization"
+)
+
+func newOrganizationCmd(out io.Writer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "organization",
+		Aliases: []string{"or"},
```

Comment:
```suggestion
		Aliases: []string{"org"},
```

nitpick: maybe `org` might be a better alias, leaving it up to you to decide